lib/events/firestoreevents/firestoreevents.go

```diff
@@ -515,6 +516,13 @@ func (l *Log) SearchEvents(fromUTC, toUTC time.Time, filter string, limit int) (
 				break
 			}
 		}
+		if events.IsRecordOff(fields) {
```

Comment:
Is this something that needs to be done manually with each back end? Or can it be hoisted out to the next level up?

(I suspect I'm missing some context here, and fully expect the answer to me "no")